go1.13/arm/Dockerfile.tmpl

```diff
@@ -3,13 +3,49 @@ ARG VERSION
 FROM ${REPOSITORY}/golang-crossbuild:${VERSION}-base
 
 RUN \
-    apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated \
+    dpkg --add-architecture arm64 \
+#    && dpkg --add-architecture armhf \
+#    && dpkg --add-architecture armel \
+    && apt-get update -qq \
+    && apt-get install -qq -y \
+      --no-install-recommends \
+      --allow-unauthenticated \
         crossbuild-essential-arm64 \
         crossbuild-essential-armel \
         crossbuild-essential-armhf \
         linux-libc-dev-arm64-cross \
         linux-libc-dev-armel-cross \
         linux-libc-dev-armhf-cross \
+        librpm-dev:arm64 \
+        libc-dev:arm64 \
+        libpopt-dev:arm64 \
+        linux-libc-dev:arm64 \
+        libxml2-dev:arm64 \
+        libxml2:arm64 \
+        libicu-dev:arm64 \
+        libicu57:arm64 \
+        icu-devtools:arm64 \
+        libsystemd-dev:arm64 \
+#        librpm-dev:armhf \
```

Comment:
we cannot install the three ARM variants, the packages conflict with each other. I have to take a look at https://wiki.debian.org/CrossCompiling to see if there is a way to make it. An alternative is to have an image for each ARM variant